src/services/checkUserName.js

```diff
@@ -0,0 +1,11 @@
+const CheckUserName = function CheckUserName(input) {
+  const user = /^[A-Za-z]\w{1,}$/;
+  if (user.test(input) && input) {
```

Comment:
Same. Also these functions serve a similar purpose and they are fairly small, they should be in the same file.

Reply:
Same as earlier.